Dockerfile

```diff
@@ -1,145 +1,166 @@
 #--------------------------------------
 # Target image to build
 #--------------------------------------
-ARG IMAGE=latest
+ARG TARGET=latest
 
 #--------------------------------------
 # Ubuntu base image to use
 #--------------------------------------
-ARG FLAVOR=latest
+ARG FLAVOR=bionic
+ARG BASE_IMAGE=${FLAVOR}
+ARG USER_NAME=ubuntu
+ARG USER_ID=1000
+ARG APP_ROOT=/usr/src/app
 
 #--------------------------------------
 # renovate rebuild trigger
 #--------------------------------------
-FROM renovate/ubuntu@sha256:b20bd58a962952580352f731606c63f621e525cc4706a4b78baf71e852b69da4 as base-latest
-FROM renovate/ubuntu:bionic@sha256:1ab4cb684944bfd76b2a29c368e4e221b796f08fb826c1a826327476cf196647 as base-bionic
-FROM renovate/ubuntu:focal@sha256:b20bd58a962952580352f731606c63f621e525cc4706a4b78baf71e852b69da4 as base-focal
+FROM ubuntu:bionic@sha256:ea188fdc5be9b25ca048f1e882b33f1bc763fb976a8a4fea446b38ed0efcbeba as latest
+FROM ubuntu:bionic@sha256:ea188fdc5be9b25ca048f1e882b33f1bc763fb976a8a4fea446b38ed0efcbeba as bionic
+FROM ubuntu:focal@sha256:703218c0465075f4425e58fac086e09e1de5c340b12976ab9eb8ad26615c3715 as focal
 
 #--------------------------------------
 # Image: base
 #--------------------------------------
-FROM base-${FLAVOR} as base
+FROM ${BASE_IMAGE} as base
 
-ARG BUILDPACK_VERSION
-LABEL org.opencontainers.image.source="https://github.com/renovatebot/docker-buildpack" \
-  org.opencontainers.image.version="${BUILDPACK_VERSION}"
-
-USER root
+ARG BUILDPACK_VERSION=custom
```

Comment:
What's the meaning of this term? Can it be removed?

Reply:
Just don't wan an empty version, but it doesn't matter. We can completly remove this and the label too. 🤷‍♂️

Reply:
Let's remove them both